forms-flow-web/src/apiManager/services/applicationServices.js

```diff
@@ -217,7 +217,7 @@ export const FilterApplications = (params, ...rest) => {
     if (id && id !== "") {
```

Comment:
an empty string is falsy, right? so we can omit these empty string checks and check for the properties instead